targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_MCU_K64F/device/TOOLCHAIN_GCC_ARM/MK64FN1M0xxx12.ld

```diff
@@ -192,15 +203,69 @@ SECTIONS
 
   .interrupts_ram :
   {
-    . = ALIGN(8);
+    . = ALIGN(4);
     __VECTOR_RAM__ = .;
     __interrupts_ram_start__ = .; /* Create a global symbol at data start */
     *(.m_interrupts_ram)     /* This is a user defined section */
     . += M_VECTOR_RAM_SIZE;
-    . = ALIGN(8);
+    . = ALIGN(4);
     __interrupts_ram_end__ = .; /* Define a global symbol at data end */
   } > m_data
+  
+  .crash_data_ram :
+  {
+    . = ALIGN(4);
+    __CRASH_DATA_RAM__ = .;
+    __CRASH_DATA_RAM_START__ = .; /* Create a global symbol at data start */
+    KEEP(*(.keep.crash_data_ram))
+    *(.m_crash_data_ram)     /* This is a user defined section */
+    . += M_CRASH_DATA_RAM_SIZE;
+    . = ALIGN(4);
+    __CRASH_DATA_RAM_END__ = .; /* Define a global symbol at data end */
+  } > m_data
+
+  /* uVisor own memory and private box memories
```

Comment:
uvisor changes?

Reply:
I think these are rebase issues, I'll fix them, Thanks for pointing them out.